ci/ray_ci/test_bisector.py

```diff
@@ -0,0 +1,20 @@
+import sys
+import pytest
+from unittest import mock
+
+from ci.ray_ci.macos_bisector import MacOSBisector
+
+
+@mock.patch("ci.ray_ci.bisector.Bisector._checkout_and_validate")
+@mock.patch("ci.ray_ci.bisector.Bisector._get_revision_lists")
+def test_run(mock_get_revision_lists, mock_checkout_and_validate):
```

Comment:
maybe some more unit tests on the bisect logic? that part is often a bit error prone.